3-fleetscope/terraform.example.tfvars

```diff
@@ -15,9 +15,9 @@
  */
 
 namespace_ids = {
-  "cb-frontend"     = "your-frontend-group@yourdomain.com",
-  "cb-accounts"     = "your-accounts-group@yourdomain.com",
-  "cb-transactions" = "your-transactions-group@yourdomain.com"
+  "cb-frontend" = "your-frontend-group@yourdomain.com",
+  "cb-accounts" = "your-accounts-group@yourdomain.com",
+  "cb-ledger"   = "your-transactions-group@yourdomain.com"
```

Comment:
nit: consistency?

```suggestion
  "cb-ledger"   = "your-ledger-group@yourdomain.com"
```

Reply:
@apeabody updated!

just a note, this doc is using the transactions namespace https://cloud.google.com/architecture/enterprise-application-blueprint/cymbal-bank

Reply:
Thanks @caetano-colin - Let me look into that page.

Reply:
Hi @caetano-colin - I've updated that page.